jest-setup.ts

```diff
@@ -29,6 +29,39 @@ if ( typeof window !== 'undefined' ) {
 nock.disableNetConnect();
 nock.enableNetConnect( 'raw.githubusercontent.com' );
 
+/**
+ * TODO: Replace this manual filter with a more robust logger that can be
+ * disabled for the testing environment. Consider enabling a lint rule that
+ * discourages direct use of `console` methods.
```

Comment:
I consider this filtering approach to be a quick solution, but a more ideal solution would be a central logger utility that manages and formats logging in a consistent manner. This might allow us to disable logs in certain contexts (e.g., test runs via a `VERBOSE=false` environment variable) and ensure the same information (e.g., helpful metadata) is included for each log.